components/brave_welcome_ui/components/wrapper/index.ts

```diff
@@ -81,6 +84,7 @@ export const Content = styled<ContentProps, 'section'>('section')`
   flex: 1;
   max-width: 580px;
   padding: 24px;
+  margin: auto;
```

Comment:
nit: I believe `margin` on an absolutely positioned element will potentially have weird side effects that could hurt maintainability in the future. Since in this case, the element is positioned absolute relative to the grandparent, but the `auto` margin is relative to the parent. Marking as a nit only, because right now it is having the desired effect since the parent and the grandparent have the same coordinates and width. It also seems tough to change since I would probably instead do this with align-items: center on the parent but absolute positioning ignores that of course. In fact, the parent flex properties seems irrelevant now. So, one way or another this nit is about cleaning up the code to only use what's needed.

As an aside, I am interested as to why absolute positioning was needed when we don't animate absolute position, but animate transform, which is compatible with regular layout.

Reply:
absolute positioning is needed for invisible content so they don't mess with the page layout. I updated this PR to fit your feedback. please re-review when you get a chance